### DISCUSSION TITLE
DAI 2024-2025 - My picocli project - Yoann Changanaqui

### DISCUSSION THREAD

**Discussion by Yoy017** [heig-vd-dai-course/heig-vd-dai-course#302] [Show and tell]
https://github.com/Yoy017/04-java-intellij-idea-and-maven

**Comment by ludelafo** [comment]
Looks good to me! Even with the signed commits, congrats!

Have you seen my other comment on your pending PR (<https://github.com/heig-vd-dai-course/.github/pull/114#pullrequestreview-2321961628>)? I have replied to your email as well yesterday, did I answer all your questions?

Please let me know if I can be of any help. :)
